### ISSUE TITLE
Another Exception and inaccessible web interface due to js load

### ISSUE THREAD

**Issue by wakeup** [owncloud/core#13369]
After today's git pull, I am unable to access to webinterface.

owncloud.log:

~~~
{"reqId":"b6937ff43fd1c0b3e705631d98e406c6","remoteAddr":"127.0.0.1","app":"index","message":"Exception: {\"Message\":\"js file not found: script:3rdparty\\\/chosen\\\/l10n\\\/tr serverroot:\\\/var\\\/www\\\/oc_git\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/templatelayout.php(153): OC\\\\Template\\\\ResourceLocator->find(Array)\\n#1 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/templatelayout.php(104): OC_TemplateLayout::findJavascriptFiles(Array)\\n#2 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/template.php(118): OC_TemplateLayout->__construct('guest', '')\\n#3 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/template\\\/base.php(103): OC_Template->fetchPage()\\n#4 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/template.php(197): OC\\\\Template\\\\Base->printPage()\\n#5 \\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/util.php(805): OC_Template::printGuestPage('', 'login', Array)\\n#6 \\\/var\\\/www\\\/oc_git\\\/lib\\\/base.php(861): OC_Util::displayLoginPage(Array)\\n#7 \\\/var\\\/www\\\/oc_git\\\/lib\\\/base.php(821): OC::handleLogin()\\n#8 \\\/var\\\/www\\\/oc_git\\\/index.php(28): OC::handleRequest()\\n#9 {main}\",\"File\":\"\\\/var\\\/www\\\/oc_git\\\/lib\\\/private\\\/template\\\/resourcelocator.php\",\"Line\":46}","level":4,"time":"2015-01-14T19:11:54+00:00"}
{"reqId":"b6937ff43fd1c0b3e705631d98e406c6","remoteAddr":"127.0.0.1","app":"PHP","message":"Uncaught exception 'Exception' with message 'js file not found: script:3rdparty\/chosen\/l10n\/tr serverroot:\/var\/www\/oc_git' in \/var\/www\/oc_git\/lib\/private\/template\/resourcelocator.php:46\nStack trace:\n#0 \/var\/www\/oc_git\/lib\/private\/templatelayout.php(153): OC\\Template\\ResourceLocator->find(Array)\n#1 \/var\/www\/oc_git\/lib\/private\/templatelayout.php(104): OC_TemplateLayout::findJavascriptFiles(Array)\n#2 \/var\/www\/oc_git\/lib\/private\/template.php(118): OC_TemplateLayout->__construct('error', '')\n#3 \/var\/www\/oc_git\/lib\/private\/template\/base.php(103): OC_Template->fetchPage()\n#4 \/var\/www\/oc_git\/lib\/private\/template.php(227): OC\\Template\\Base->printPage()\n#5 [internal function]: OC_Template::printExceptionErrorPage(Object(Exception))\n#6 {main}\n  thrown at \/var\/www\/oc_git\/lib\/private\/template\/resourcelocator.php#46","level":3,"time":"2015-01-14T19:11:54+00:00"}
~~~

config:

~~~
<?php
define('DEBUG',true);
$CONFIG = array (
  'instanceid' => 'x',
  'passwordsalt' => 'xx',
  'secret' => 'xx',
  'trusted_domains' => 
  array (
    0 => 'localhost',
    1 => 'xxx',
  ),
  'datadirectory' => '/var/www/oc_git/data',
  'overwrite.cli.url' => 'http://localhost/oc_git',
  'dbtype' => 'sqlite3',
  'version' => '8.0.0.2',
  'installed' => true,
  'mail_smtpmode' => 'php',
  'apps_paths' => 
  array (
    0 => 
    array (
      'path' => '/var/www/oc_git/apps',
      'url' => '/apps',
      'writable' => false,
    ),
    1 => 
    array (
      'path' => '/var/www/oc_git/apps2',
      'url' => '/apps2',
      'writable' => false,
    ),
    2 => 
    array (
      'path' => '/var/www/oc_git/appsstore',
      'url' => '/appsstore',
      'writable' => true,
    ),
  ),
  'theme' => '',
  'maintenance' => false,
);

~~~

3rdparty log shows the latest commit is: 05f91884356103530090d81503250111da50bdb4
however doing git submodule update, detaches it from master.

Using php: 5.5.12-2ubuntu4.1 on Apache/2.4.10 (Ubuntu) 64-bit

**Comment by DeepDiver1975** [comment]
@Raydiation can I ask you to take a look? THX

**Comment by Raydiation** [comment]
@DeepDiver1975 can not reproduce

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 can not reproduce

I can with calendar being enabled :wink: 

@wakeup there is a pr in the calendar repo which has to be applied to make it work again: https://github.com/owncloud/calendar/pull/671

**Comment by DeepDiver1975** [comment]
fixed as owncloud/calendar#671 was merged

**Comment by wakeup** [comment]
Thanks, works :).
